nextflow_schema.json

```diff
@@ -580,5 +472,14 @@
         {
             "$ref": "#/definitions/institutional_config_options"
         }
-    ]
+    ],
+    "properties": {
+        "clusterOptions": {
+            "type": "string",
+            "description": "Arguments passed to Nextflow clusterOptions.",
```

Comment:
```suggestion
            "description": "Arguments passed to Nextflow cluster_options.",
```

Reply:
In the description, it should remain `clusterOptions` because it is a [nextflow directive](https://www.nextflow.io/docs/latest/process.html#process-clusteroptions)
It's a bit confusing that the pipeline parameter is named the same, I know.

It needs to be changed to snake_case in the `nextflow.config` and the docs though

Reply:
I will add this to `nf-core lint` so that `clusterOptions` is not allowed 👍🏻

Reply:
How far are we here? I'm a bit lost on what needs to be done now

Reply:
@apeltzer I think this is good to go once the switch from `params.clusterOptions` -> `params.cluster_options` is made.
I would like to get this in before the 1.6 release, mostly because the Schema validation prints a warning if the parameter name is `CamelCase`

@tmuylder Would be great if you could find some time to :taco: this up, so we can have a release later :+1:

Reply:
Looking at this again - the problem with this parameter is that it's a bit of a catch-22 situation.. It's used extensively in the centralised configs: https://github.com/nf-core/configs/search?q=clusteroptions

So updating in pipelines will mean that they need updating. And updating them will break them for any pipelines that haven't updated yet.. I think this is why we haven't done it yet.

But if linting is failing then yes that may be good enough motivation to do it. But should probably go out in a template sync to try to get all pipelines to change at once.